### ISSUE TITLE
Receive Inventory - Added fields don’t show up in the dropdown immediately

### ISSUE THREAD

**Issue by jackkrooss** [tulip/ppe-logistics#140]
Simon Bass

**Comment by ianslai** [comment]
To clarify: Does this mean that when you type something into "Create new variant" or "Add new supplier", it should be added immediately to the dropdowns to the left?  (If so I would be hesitant because that means you can easily create spurious Variants/Suppliers by entering text into those boxes without tapping some kind of confirmation button.)

**Comment by jackkrooss** [comment]
No, it means that after you submit the form (Receive Inventory) that the
dropdown is not updated until you bounce out of the form to WN (workspace
navigation) and back in

**Comment by ianslai** [comment]
Thanks for the clarification @jackkrooss .  I've put in a fix that refreshes both Suppliers and Variants on the Receive Inventory screen (Apps to Share / Production Worker / Receive Inventory).
